biggus/__init__.py

```diff
@@ -2125,8 +2187,20 @@ def _sliced_shape(shape, keys):
 
     """
     sliced_shape = []
-    # TODO: Watch out for more keys than shape entries.
```

Comment:
I've taken a punt at this function based on previous work I've done. #124 refers.